launch/dual_ekf_navsat_example.launch.py

```diff
@@ -0,0 +1,53 @@
+from launch import LaunchDescription
```

Comment:
Wait, you can't just remove someone elses license when this was based on it. Please readd the OSRF license header.

Reply:
would OSRF really agree that I do bunch of unrelated changes to their code and not implying that on the license ? 
I will do what you are asking here but
with all due respect,  I hate to post things as below but I have no choice here; 

Question : What to do a File with Apache 2.0 and with modifications 
```
Section 4 of the Apache License 2.0 is quite clear on what you must do when you distribute the changed file:

    You must not remove the existing copyright claim (the one by 'the Best Company in the World')

    You must make it clear the the file has been changed. The easiest way is to simply add your copyright after the original ones:

    Modifications copyright (C) 2013 <your company/name>

    If you did the modifications on behalf of your company, then that is in most cases also the name that you must put on the copyright notice
```
link to source of above information; https://softwareengineering.stackexchange.com/questions/220068/file-with-apache-2-0-and-my-modifications

Reply:
My aim isn't to get into any argument about it, but my rule of thumb on this is to be generous with copyrights that exist currently when copying files around but "significant" changes have to be added to add a new copyright marker on the header. "Significant" is ill-defined, but I usually say its something like a strong refactor or adding substantial new code to the file.  

I would qualify launch file work as not really that significant unless you were to add in a bunch of declared parameters or significantly refactor it. You still own your own copyright for this work (or your employer), listing it here doesn't gain or deny you anything. 

Either way, I don't care that much. The "most-wrong" answer here is to simply delete it along with the OSRF statement. The statement your linked to there is a clear sense that the file changed (e.g. Apache doesn't specify anything about version control, we have git and there's blaming on git so we know exactly who's modified things from that history and generated change logs on release).

Reply:
If you feel strongly about having a copyright on this launch file, go for it and I'll approve it this time. But we might revisit this conversation at a later date in other PRs if you're only lightly modifying other people's work.

Reply:
I do agree with you that this is a simple launch file and does not add any novelty I added the second license because of my concern here 
 > would OSRF really agree that I do bunch of unrelated changes to their code and not implying that on the license ?

since I am not affiliated with OSRF.
if that's not an issue then in the current form this PR should be ready to go. Apologies for any inconvenience.